### PR TITLE
Task-49591: The mention updated is displayed in the article's details even if not updated.

### PR DESCRIPTION
**Pull request by ahmadmido**
Before this fix, even if the news isn't yet updated or modified, the mention updated date is displayed in the article's details.
This fix will ensure that the mention updated date is displayed only if the news is updated.

### CODE REVIEW

File: webapp/src/main/webapp/news-details/components/ExoNewsDetailsBody.vue [hunk 1]
```diff
@@ -150,7 +150,7 @@ export default {
       return this.news && this.news.title;
     },
     showUpdateInfo() {
-      return this.updatedDate || (this.news && this.news.updatedDate && this.news.updatedDate  !== 'null');
+      return this.news && (this.news.updateDate && this.news.updateDate !== 'null' && this.news.publicationDate && this.news.publicationDate!== 'null') &&  this.news.updateDate.time > this.news.publicationDate.time;
```
Comment (azayati) [inline webapp/src/main/webapp/news-details/components/ExoNewsDetailsBody.vue]:
```suggestion
      return this.news && this.news.updateDate && this.news.updateDate !== 'null' && this.news.publicationDate && this.news.publicationDate!== 'null' && this.news.updateDate.time > this.news.publicationDate.time;
```

File: webapp/src/main/webapp/news-details/components/ExoNewsDetailsTime.vue [hunk 2]
```diff
@@ -76,7 +76,7 @@ export default {
       return this.news && `${eXo.env.portal.context}/${eXo.env.portal.portalName}/profile/${this.news.updater}`;
     },
     showUpdateInfo() {
-      return this.updatedDate || (this.news && this.news.updatedDate && this.news.updatedDate  !== 'null');
+      return this.news && (this.news.updateDate && this.news.updateDate !== 'null' && this.news.publicationDate && this.news.publicationDate!== 'null') &&  this.news.updateDate.time > this.news.publicationDate.time;
```
Comment (azayati) [inline webapp/src/main/webapp/news-details/components/ExoNewsDetailsTime.vue]: not used here to be deleted IMHO cc @hassenelkamel-exo

**Review by azayati — CHANGES_REQUESTED** [review]
see review

File: webapp/src/main/webapp/news-details/components/ExoNewsDetailsTime.vue (hunk 2 above)
Comment (hassenelkamel-exo) [reply webapp/src/main/webapp/news-details/components/ExoNewsDetailsTime.vue]: yes, you can remove it

**Review by azayati — APPROVED** [review]
LGTM after QA validation

**Pull request merged**